src/renderer/components/table/table-cell.tsx

```diff
@@ -15,6 +15,7 @@ export interface TableCellProps extends React.DOMAttributes<HTMLDivElement> {
   isChecked?: boolean; // mark checkbox as checked or not
   renderBoolean?: boolean; // show "true" or "false" for all of the children elements are "typeof boolean"
   sortBy?: TableSortBy; // column name, must be same as key in sortable object <Table sortable={}/>
+  showWithColumn?: string //visibility state of the column depends on another one state
```

Comment:
This should have a comment saying that it should be the same as the `className` of the other column.

Reply:
Extended this comment, resolved conflicts